cmd/state/main.go

```diff
@@ -147,12 +148,14 @@ func run(args []string, out output.Outputer) (int, error) {
 	// Run the actual command
 	cmds := cmdtree.New(primer.New(pj, out, authentication.Get(), prompter, sshell, conditional))
 
-	var skipChecks bool
-	child := cmds.Command().Find(args)
-	if child != nil {
-		skipChecks = child.SkipChecks()
+	var child *captain.Command
+	if len(args) != 0 {
```

Comment:
`args` will never be 0, since `args[0]` is always the executable that was invoked.